src/v2/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -0,0 +1,145 @@
+import { BorderedRadio, Flex, RadioGroup, Text } from "@artsy/palette"
+import { useEffect, useState } from "react"
+import { OfferInput } from "v2/Apps/Order/Components/OfferInput"
+import { compact } from "lodash"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { PriceOptions_artwork } from "v2/__generated__/PriceOptions_artwork.graphql"
+import { appendCurrencySymbol } from "../Utils/currencyUtils"
+
+export interface PriceOptionsProps {
+  setValue: (value: number) => void
+  onFocus: () => void
+  showError?: boolean
+  artwork: PriceOptions_artwork | null | undefined
```

Comment:
You don't need to specify `null | undefined` here, the `PriceOptions_artwork`, artwork in this context will be always defined(you can't offer to an undefined artwork) but the properties might be null see auto-generated graphql file